### PR TITLE
Wooc 385

### PR DESCRIPTION
**Pull request by issam-payplug**

### CODE REVIEW

**Review by PPmmesquita — APPROVED** [review]

**Review by PPmmesquita — CHANGES_REQUESTED** [review]
please check the comments I've left on gitlab rebuild and recommit the build.
translations are good (/)

**Comment by issam-payplug** [comment]
> please check the comments I've left on gitlab rebuild and recommit the build. translations are good (/)

Done

File: languages/payplug-en_US.po [hunk 1]
```diff
@@ -14,6 +14,9 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: Poedit 3.0.1\n"
 
+msgid "Advanced Settings"
```
Comment (GhDj) [inline languages/payplug-en_US.po:17]: "Advanced Settings" better use snake case for translations ids

File: languages/payplug.pot [hunk 2]
```diff
@@ -715,3 +715,49 @@ msgstr "You don't have access to this feature yet. To activate American Express,
...
+msgid "payplug_thresholds_oney_title"
+msgstr ""
+
+msgid "payplug_thresholds_oney_description"
+msgstr ""
+
+msgid "and"
+msgstr ""
+
+
```
Comment (GhDj) [inline languages/payplug.pot:752]: "Advanced Settings" italian translation missing

**Review by GhDj — CHANGES_REQUESTED** [review]

File: src/Admin/Vue.php [hunk 3]
```diff
@@ -265,24 +265,27 @@ public function payplug_section_paylater($active = false) {
...
+						"link_know_more" => Component::link(__( 'payplug_know_more_label', 'payplug' ), "https://support.payplug.com/hc/fr/articles/4408142346002", "_blank"),
 					],
 					"sandbox" => [
-						"description"      => __( 'payplug_section_applepay_payment_description', 'payplug' ),
-						"link_know_more" => Component::link(__( 'payplug_section_bancontact_payment_know_more_label', 'payplug' ),"https://support.payplug.com/hc/en-gb/articles/360013071080", "_blank"),
+						"description"      => __( 'payplug_section_paylater_description_oney', 'payplug' ),
+						"link_know_more" => Component::link(__( 'payplug_know_more_label', 'payplug' ), "https://support.payplug.com/hc/fr/articles/4408142346002", "_blank"),
+					],
+					"advanced" => [
+						"description" => __( 'Advanced Settings', 'payplug' ),""
```
[the remark this answers is not in the document]
Comment (issam-payplug) [reply src/Admin/Vue.php:276]: Done

File: languages/payplug-fr_FR.po [hunk 4]
```diff
@@ -46,7 +46,7 @@ msgstr "Activer le mode debug"
 
 #: src/Gateway/PayplugGateway.php:357
 msgid "Advanced Settings"
```
[the remark this answers is not in the document]
Comment (issam-payplug) [reply languages/payplug-fr_FR.po:48]: Done

File: languages/payplug-en_US.po (hunk 1 above)
Comment (issam-payplug) [reply languages/payplug-en_US.po:17]: Done

**Review by PPmmesquita — APPROVED** [review]

**Pull request merged**